sdk/mediaservices/Azure.ResourceManager.Media/CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2022-09-14)
 
 ### Features Added
 
+- Upgrade to `package-metadata-2022-08`
+
 ### Breaking Changes
 
+- Renamed `TransformOutputsPriority` to `MediaTransformOutputsPriority`.
+- Renamed `ContentKeyPolicyPreference` to `ContentKeyPolicyOption:`.
```

Comment:
Please remove the extra colon at the end of this line.

Reply:
fixed